thunder/images/images.py

```diff
@@ -549,6 +549,10 @@ def map_as_series(self, func, value_size=None, block_size='150'):
             func. If not supplied, will be automatically inferred for an extra
             computational cost.
 
+        dtype : str, optional, default = None
+            dtype of one-dimensional ndarray resulting from application of func.
+            if not supplied it will be automatically inferred for an extra computational cost.
```

Comment:
Capitalize (I only nitpick since this could get auto-generated to documentation)